addons/l10n_in_hr_payroll/models/l10n_in_hr_payroll.py

```diff
@@ -109,7 +109,7 @@ def cancel_sheet(self):
 
     @api.onchange('company_id')
     def _onchange_company_id(self):
-        self.bank_id = self.company_id.partner_id.bank_ids and self.company_id.partner_id.bank_ids[0].bank_id.id or False
+        self.bank_id = self.company_id.partner_id.bank_ids and self.company_id.partner_id.bank_ids[0].bank_id and self.company_id.partner_id.bank_ids[0].bank_id.id or False
```

Comment:
Put directly `.id` and remove the new `and...` and the `or False` part.

Reply:
In brief, revert the change